spec/models/server_role_spec.rb

```diff
@@ -55,7 +55,11 @@
         web_services,Web Services,0,false,region
       CSV
 
-      allow(File).to receive(:open).and_return(StringIO.new(@csv))
+      allow(File).to receive(:exist?).and_call_original
+      allow(File).to receive(:exist?).with(a_string_matching(/content\/server_roles.csv/)).and_return(false)
```

Comment:
Had to switch to using `File.exist?(file.to_s)` to make it simpler to stub there here, open to other suggestions on mocking `Pathname#exist?` though